### ISSUE TITLE
Image Seems to Broken for HP Provider

### ISSUE THREAD

**Issue by TerryHowe** [fog/fog#3420]
My guess is that HP public cloud compute no longer supports listing images.  It should be using the image service (glance) anyway.  Anyway, compute (nova) is 502ing or 500ing image requests.

**Comment by geemus** [comment]
Thanks for the heads up. Would you be up for digging a bit deeper to try and work on a fix?

**Comment by TerryHowe** [comment]
It looks somewhat complicated and I'm spread a little thin at the moment.  Ideally, I'd like to see the HP provider use the OpenStack provider for cases like this.  The HP provider should only really be extending OpenStack for extensions like DNSaaS, LBaaS, etc.

**Comment by geemus** [comment]
Agreed, would be great to see a more unified front there, but I think the efforts around that have lost a bit of steam lately. I gather we are all spread a bit thin unfortunately.
